src/select-menu/src/Option.js

```diff
@@ -1,14 +1,33 @@
 import React, { memo, forwardRef } from 'react'
+import cx from 'classnames'
 import PropTypes from 'prop-types'
+import { useStyleConfig } from '../../hooks'
 import { Image } from '../../image'
 import { Pane } from '../../layers'
 import TableRow from '../../table/src/TableRow'
 import TextTableCell from '../../table/src/TextTableCell'
-import { useTheme } from '../../theme'
+
+export const pseudoSelectors = {
+  _active: '&[aria-current="true"]:active, &[data-isselectable="true"]:active',
+  _before: '&:before',
+  _disabled: '&[disabled]',
+  _focus: ':focus',
+  _hover: ':hover',
+  _isSelectable: '&[data-isselectable="true"]',
+  _selected: '&[aria-current="true"]'
+}
+
+const internalStyles = {
+  alignItems: 'center',
+  display: 'flex'
+}
+
+const emptyObject = {}
 
 const Option = memo(
   forwardRef(function Option(props, ref) {
     const {
+      className,
```

Comment:
Do we need to add the additional classname here? I don't think we export `<Option />` in EG, so consumers can't style this initially.

Reply:
that makes sense, i will remove this